sys/net/gnrc/network_layer/sixlowpan/gnrc_sixlowpan.c

```diff
@@ -216,7 +214,10 @@ static void _send(gnrc_pktsnip_t *pkt)
 
     hdr = pkt2->data;
     iface = gnrc_sixlowpan_netif_get(hdr->if_pid);
-    datagram_size = gnrc_pkt_len(pkt2->next);
+#ifdef MODULE_GNRC_SIXLOWPAN_FRAG
+    /* datagram_size: pure IPv6 packet without 6LoWPAN dispatches or compression */
+    size_t datagram_size = gnrc_pkt_len(pkt2->next);
```

Comment:
Please replace this variable wit the function call in https://github.com/RIOT-OS/RIOT/pull/3974/files#diff-e87e12a912a7eb2cbb8a16f4ebeccabaR281 (and other places I might have overlooked).